applications/app/controllers/CrosswordsController.scala

```diff
@@ -107,19 +119,7 @@ object CrosswordSearchController extends Controller with ExecutionContexts {
 
   def lookup() = Action.async { implicit request =>
     lookupForm.bindFromRequest.get match {
```

Comment:
I know this isn't in your change, but I am surprised to see this...
The `case _` will never happen bellow because `get` throws an exception from `Form[T]`.
I think we should probably be using `Form.fold` here, [see example](https://github.com/guardian/frontend/blob/master/applications/app/controllers/EmailController.scala#L124)